app/workers/application_worker.rb

```diff
@@ -0,0 +1,50 @@
+require 'prx_access'
+
+class ApplicationWorker
```

Comment:
This class reimplements some of the `announce` gem delegation logic, to make it easier to move those jobs to shoryuken workers.

Reply:
I also put in queue naming logic, which cleans up the workers a bit too